src/portal-popper.jsx

```diff
@@ -35,6 +35,11 @@ class PortalPopper extends Component {
     flipped: false,
   }
 
+  constructor (props) {
```

Comment:
Instead of binding to this in the constructor, use the `method = () =>` form instead. See the `_updateData` method for an example.